### ISSUE TITLE
[TD] 1 pixel tall gaps in DOS mode sidebar

### ISSUE THREAD

**Issue by hazelnot** [TheAssemblyArmada/Vanilla-Conquer#756]
Not sure if this is a regression or just something I hadn't noticed before, but there seem to be two 1 pixel tall gaps in the DOS mode sidebar, above the power meter and the cameo strips.

Vanilla Conquer DOS mode:
![image](https://user-images.githubusercontent.com/10992973/161078217-031ea694-8a6d-46a5-8b1a-22be0ab82ec5.png)

Promo image and I assume also what it looks like in the DOS version:
![image](https://user-images.githubusercontent.com/10992973/161079610-5ecf6d18-fb61-40bc-baaf-0c2990a0ca3a.png)

There's a gap on the left about 9 pixels wide, and a pixel-sized gap on the right side.

**Comment by Nyerguds** [comment]
Definitely something not right there. This is the Vanilla Conquer DOS mode shot you have there, overlayed on an actual DOSBox screenshot:
![ccdos_sidebar](https://user-images.githubusercontent.com/1840281/162392816-34c33a47-db83-4457-9e80-c7a3a0edb795.gif)

The icons seem misaligned horizontally, too.

**Comment by hazelnot** [comment]
Huh, the power meter seems slighly off here too, the shade of grey on the lines is different and they seem to also be misaligned in DOS mode, even though it's drawn in a completely different way lmao

**Comment by Nyerguds** [comment]
Of course, I don't have the full context of your screenshot. For all I know, the radar and sell buttons could already be misaligned. I'd need to see a full sidebar to compare.

**Comment by hazelnot** [comment]
![image](https://user-images.githubusercontent.com/10992973/162536206-c21d5aba-c4aa-4a6a-9fd7-241585333664.png)

Ask and you shall receive

**Comment by giulianobelinassi** [comment]
Here is a list of things:
* Cameo buttons are one pixel to the left.
* Cameo columns spacement is 3 (or 4) pixels to the left
* Power gauge meter is incorrect color, and incorrect position
* Powebar is also not transparent.

**Comment by OmniBlade** [comment]
@hazelnot Can you confirm #1000 fixes this?

**Comment by hazelnot** [comment]
Partially. the 1px gap is gone, but the rest of the issues Giuliano brought up still apply
